### PR TITLE
LPS-69107 Regen

### PR DESCRIPTION
**Pull request by marcellustavares**

### CODE REVIEW

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
@linolaoj There's a conflict with master on the system.packages.extra.mf. Can you fix it?

Thanks

File: modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp [hunk 1]
```diff
@@ -294,6 +294,66 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
...
+												map.on('positionChange', geolocationField.onPositionChange, geolocationField);
+											}
+										);
+									},
+
+									onPositionChange: function(event) {
+										var instance = geolocationField;
+
+										var location = event.newVal.location;
+										var locationString = JSON.stringify(
```
Comment (brunobasto) [inline modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp]: @linolaoj The indentation here is very messy. Can you fix it too?

File: modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp [hunk 2]
```diff
@@ -294,6 +294,66 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
...
+									name='<%= "ExpandoAttribute--" + name +"--" %>'
+								/>
+							</div>
+
+							<aui:script use="aui-base, liferay-map-base">
+
+								var inputName = "<%= portletDisplay.getNamespace()+"ExpandoAttribute--" + name + "--" %>";
+
+								var geolocationField = {
+
```
Comment (brunobasto) [inline modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp]: No need for this empty line.

File: modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp [hunk 3]
```diff
@@ -294,6 +294,66 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
...
+										var location = event.newVal.location;
+										var locationString = JSON.stringify(
+																	{
+																		latitude: location.lat,
+																		longitude: location.lng
+																	}
+																)
+
+										var inputNode = A.one('[name="'+inputName+'"]');
```
Comment (brunobasto) [inline modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp]: Put spaces between the '+' sign and the variables

File: modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp [hunk 4]
```diff
@@ -294,6 +294,66 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
...
+								<liferay-map:map-display
+									geolocation="<%= true %>"
+									latitude='<%= geolocationJson.getDouble("latitude", -33.8688) %>'
+									longitude='<%= geolocationJson.getDouble("longitude", 151.2195) %>'
+									name='<%= "ExpandoAttribute--" + name +"--" %>'
+								/>
+							</div>
+
+							<aui:script use="aui-base, liferay-map-base">
+
```
Comment (brunobasto) [inline modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp]: No need for this empty line.

**Comment by brunobasto** [comment]
@linolaoj I'm closing this one for now. Can you fix it and resend?

Thanks,

**Comment by linolaoj** [comment]
Thank you @brunobasto, 
I"ll make the changes and resend.

**Comment by linolaoj** [comment]
Here it is https://github.com/brunobasto/liferay-portal/pull/801
Thanks!

**Pull request closed without merging**